python/sort_number.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+
+#This function will sort a given sequence of numbers
+#if value in the list of numbers is not interger or float, program will exit
+def sort_numbers(numbers):
+    for value in numbers:
+        check_value_number = value.replace('.', '', 1).isdigit()
+        if not check_value_number:
+            print("Error: " ,value, " is not integer or float.")
```

Comment:
As I mentioned in my last comments, no need for the spaces inside the quotes before and after the variable if using commas. Python adds the spaces automatically.
```suggestion
            print("Error:", value, "is not integer or float.")
```